relay-server/src/services/project.rs

```diff
@@ -1126,21 +1126,51 @@ impl Project {
         outcome_aggregator: Addr<TrackOutcome>,
         buckets: Vec<Bucket>,
     ) -> Option<(Scoping, ProjectMetrics)> {
+        self.check_buckets_inner(outcome_aggregator, buckets)
```

Comment:
I'd probably write this with a single match unwrapping the result

```rs
match self.check_buckets_inner(outcome_aggregator, buckets) {
    Ok(r) => return Some(r),
    Err(CheckBucketsError::ProjectExpired(len)) => relay_log::error!(...),
    Err(CheckBucketsError::ProjectDisabled(len)) => ...,
    ...
}

None
```

Reply:
I like the suggestion from David as well.

Reply:
Done!